src/mlte/measurement/external_measurement.py

```diff
@@ -0,0 +1,31 @@
+"""
+Base class for measurements calculated by external functions.
+"""
+
+from __future__ import annotations
+
+from .result import Result
+from .measurement import Measurement
+
+
+class ExternalMeasurement(Measurement):
+    def __init__(self, identifier: str, result_type: type):
+        """
+        Initialize a new ExternalMeasurement measurement.
+
+        :param identifier: A unique identifier for the instance
+        :type identifier: str
+        :param result_type: The type of the Result this measurement will return.
+        :type result_type: Type
+        """
+        super().__init__(self, identifier)
+        if not issubclass(Result, result_type):
```

Comment:
Thank you for adding this explicit runtime type-check!

Reply:
No problem!